tests/lib/ignored-paths.js

```diff
@@ -45,6 +45,17 @@ describe("IgnoredPaths", function() {
             assert.lengthOf(ignoredPaths.patterns, 0);
         });
 
+        it("should accept an array for options.ignorePattern", function() {
+            var ignorePattern = ["a", "b"];
+
+            var ignoredPaths = IgnoredPaths.load({
+                ignore: false,
+                ignorePattern: ignorePattern
+            });
+            assert.ok(ignorePattern.every(function(patt) {
+                return ignoredPaths.patterns.indexOf(patt) >= 0;
```

Comment:
It's not a typo if that's what you mean. I commonly abbreviate "pattern" as "patt" (each of these being a glob pattern). But if you find it unclear it can be `pattern|glob` or whatever you prefer.